docs/guides/core-concepts/retry-ability.mdx

```diff
@@ -201,8 +198,9 @@ Cypress tries to act like a human user would using the browser.
 - Is the element behind another element?
 - Does the element have the `disabled` attribute?
 
-Actions - such as `.click()` - automatically wait until multiple built-in
-assertions like these pass, and then it will attempt to click just once.
+Action commands - such as `.click()` - automatically wait until multiple
```

Comment:
This is the first mention of action commands, also missing link

Reply:
Link added. I haven't bolded it though, randomly bolded words are a pet peeve of mine.